Makefile

```diff
@@ -116,6 +116,9 @@ L3_SRCDIR   := $(SRCDIR)
 L3_INCDIR   := $(INCDIR)
 USE_CASES   := use-cases
 
+# Name of L3 dump utility that unpacks L3 log-entries
+L3_DUMP                 := l3_dump.py
```

Comment:
No major logic-change. Just use this symbol in this file rather than the hard-coded Python tool's name.